views/js/pciCreator/dev/audioRecordingInteraction/runtime/js/recorder.js

```diff
@@ -137,6 +180,10 @@ define([
 
             analyser.getByteFrequencyData(frequencyArray);
 
+            if (frequencyArray.length && isIOSDevice(frequencyArray)) {
```

Comment:
u dont need anymore to pass the frequencyArray

Reply:
Done.